scripts/start.sh

```diff
@@ -7,13 +7,25 @@ set -x
 HOSTNAME=`hostname`
 gosu root chmod a+wrx /etc
 gosu root chmod a+wrx /tmp
-gosu root chmod a+wrx /var/spool/cwl
+
 export TMPDIR=/tmp
-export HOME=/var/spool/cwl
+env
```

Comment:
Left in for debugging?

Reply:
Indeed, there are already lines like this elsewhere. As there are sudo's in different places, environment variables change, keeping it can indeed help with debugging when things fail. Also, keep in mind we are also testing the workflow execution on other platform like Seven Bridges' CGC, current user, environment variables can be different than running it using cwltool.